app/packages/state/src/recoil/selectors.ts

```diff
@@ -142,6 +142,20 @@ export const datasetAppConfig = graphQLSyncFragmentAtom<
   }
 );
 
+export const getDisableFrameFiltering = selector<boolean>({
```

Comment:
Current pattern is to give selectors the same name as their `key`
```suggestion
export const disableFrameFiltering = selector<boolean>({
```